app/controllers/sessions_controller.rb

```diff
@@ -19,4 +19,13 @@ def create
       render :new
     end
   end
+
+  def destroy
+    session.delete(:user_id)
+    session.delete(:cart)
```

Comment:
unsure of whether or not we should be saving these session or if they are being saved for when use logs in again

Reply:
Depending on what the user story calls for at this moment, deleting is alright. Perhaps we'll have to revisit this, and change it later.